feat: include code analyzer in expanded pack (#5702)
forcedotcom/salesforcedx-vscode: packages/salesforcedx-vscode-expanded/README.md

- [Salesforce Lightning Design System (SLDS) Validator](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforce-vscode-slds)
  This extension (`salesforcedx-vscode-slds`) simplifies working with the Salesforce Lightning Design System (SLDS). It provides code completion, syntax highlighting and validation with recommended tokens and utility classes.
- [Einstein for Developers (Beta)](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-einstein-gpt) This extension (`salesforcedx-einstein-gpt`) uses generative AI to make Salesforce development in Visual Studio Code richer with features such as natural language to code generation, inline autocompletion for Apex and LWC code, and test case generation for Apex classes.

> - [Salesfoce Code Analyzer](https://marketplace.visualstudio.com/items?itemName=salesforce.sfdx-code-analyzer-vscode) This extension (`sfdx-code-analyzer-vscode`) scans your code against multiple rule engines to produce lists of violations and improve your code.

Reviewer: Do we want to update references in our doc that calls it as Beta since it will be GA'ed this Tuesday?

Author: @diyer updated

Reviewer 2: Updates made to files in the /docs folder will no longer be visible to customers. You can delete that change.  I am making the update in the sfdocs repo.

Reviewer 2: @jshackell-sfdc

> - [Salesfoce Code Analyzer](https://marketplace.visualstudio.com/items?itemName=salesforce.sfdx-code-analyzer-vscode) This extension (`sfdx-code-analyzer-vscode`) scans your code against multiple rule engines to produce lists of violations and improve your code.

Reviewer 3: ```suggestion
- [Salesfoce Code Analyzer](https://marketplace.visualstudio.com/items?itemName=salesforce.sfdx-code-analyzer-vscode) This extension (`sfdx-code-analyzer-vscode`) scans your code using multiple rule engines to produce lists of violations that you can use to improve your code.
```

> - [Salesfoce Code Analyzer](https://marketplace.visualstudio.com/items?itemName=salesforce.sfdx-code-analyzer-vscode) This extension (`sfdx-code-analyzer-vscode`) scans your code using multiple rule engines to produce lists of violations that you can use to improve your code.

Reviewer 4: ```suggestion
- [Salesforce Code Analyzer](https://marketplace.visualstudio.com/items?itemName=salesforce.sfdx-code-analyzer-vscode) This extension (`sfdx-code-analyzer-vscode`) scans your code using multiple rule engines to produce lists of violations that you can use to improve your code.
```
